main.go

```diff
@@ -98,6 +98,28 @@ func countTotalProcMem(processes rac.RACQuery) (float64, error) {
 	return float64(total), nil
 }
 
+func createInfobaseNameMap(infobases rac.RACQuery) map[string]string {
+	infoName := make(map[string]string)
+	for _, infobase := range infobases.Records {
```

Comment:
The "infobase" occurs way to much here for readability =) We're iterating over `.Records`, so I suggest we just name it accordingly: `for _, records := ...`